src/MerQure.RbMQ/Config/MerQureConnection.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+
+namespace MerQure.RbMQ.Config
+{
+    public class MerQureConnection
+    {
+        public const string DefaultConnectionString = "amqp://guest:guest@localhost:5672/";
+        private const string AutomaticRecoveryEnabledPropertyName = "automaticRecoveryEnabled";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused private field 'AutomaticRecoveryEnabledPropertyName'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1144)